src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -57,6 +58,30 @@ namespace AppInstaller::CLI::Workflow
             }
         }
 
+        bool ShouldErrorForUnsupportedArgument(UnsupportedArgumentEnum arg)
+        {
+            switch (arg)
+            {
+            case UnsupportedArgumentEnum::Location:
+                return true;
+            default:
+                return false;
+            }
+        }
+
+        AppInstaller::CLI::Argument GetUnsupportedArgument(UnsupportedArgumentEnum arg)

```

Comment:
I don't think this works; the command might use that `Args::Type` but have a different `Argument`.  You would need to search the commands arguments to find the one that matches.